resources/providers/awslib/ec2/ebs_encryption.go

```diff
@@ -0,0 +1,41 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package ec2
+
+import (
+	"fmt"
+
+	"github.com/elastic/cloudbeat/resources/fetching"
+)
+
+type EBSEncryption struct {
+	Enabled bool
+	region  string
+}
+
+func (e EBSEncryption) GetResourceArn() string {
+	return fmt.Sprintf("ebs-encryption-by-default-%s", e.region)
```

Comment:
Multi AWS accounts will lead to "collision".
Let's add the account id.

```suggestion
	return fmt.Sprintf("default-ebs-encryption-%s-%s", e.account, e.region)
```